mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -3775,14 +3775,9 @@ resource "google_container_cluster" "with_maintenance_exclusion_window" {
 		start_time = "%s"
 		end_time = "%s"
 	}
-	maintenance_exclusion {
```

Comment:
Why remove the second? A cluster can have multiple active exclusions.

Reply:
That test has two steps. The first step contains two `maintenance_exclusion` already and the 2nd step is for `Update`(2->1)  (as you suggested below) while other updating features remain

Reply:
Thanks, yep! Sorry I'd misread this.